src/RadiumVariables.js

```diff
@@ -0,0 +1,23 @@
+/*
+  Without SCSS / CSS I lose my variables so I made this object
+	to be able to still use variables with Radium.
+*/
+
+const RadiumVars = {
```

Comment:
Before adding Radium I was using SCSS to do my styling and thus used variables throughout the project.  I made this file as a replacement for the SCSS file and import it where I need to use the variables.